x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/transactions/breakdown.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import archives from '../constants/archives_metadata';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const apmApiClient = getService('apmApi');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
+  const transactionType = 'request';
+
+  describe('Breakdown', () => {
```

Comment:
Nit: Also here we can remove the 'Breakdown' from the next line and `'when data is not loaded'` wdyt?

Reply:
done!